.github/workflows/release-chart.yml

```diff
@@ -0,0 +1,41 @@
+name: Release metrics adapter chart
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  # Sometimes chart-releaser might fetch an outdated index.yaml from gh-pages, causing a WAW hazard on the repo
+  # This job checks the remote file is up to date with the local one on release
+#  validate-gh-pages-index:
+#    runs-on: ubuntu-latest
+#    steps:
+#      - name: Checkout
+#        uses: actions/checkout@v2
+#        with:
+#          ref: gh-pages
+#      - name: Download remote index file and check equality
+#        run: |
+#          curl -vsSL https://newrelic.github.io/newrelic-k8s-metrics-adapter/index.yaml > index.yaml.remote
+#          LOCAL="$(md5sum < index.yaml)"
+#          REMOTE="$(md5sum < index.yaml.remote)"
+#          echo "$LOCAL" = "$REMOTE"
+#          test "$LOCAL" = "$REMOTE"
+
+  release:
```

Comment:
since we have "docker-release" I would call it "chart-release"

Reply:
As it is the only `release` in this pipeline I didn't bother. The file name is `release-chart.yaml` yet.

Reply:
![Screen Shot 2022-03-22 at 3 06 43 PM](https://user-images.githubusercontent.com/43335750/159500224-e8ea045f-f318-4652-9cd4-b00b64aa8223.png)

Just because of this, but just leave it like that if you think that is reduntant

Reply:
That is a reminiscence of when I have the integration and chart release in the same pipeline. I did an interactive rebase to get rid of it.